bgpd/bgp_route.c

```diff
@@ -14758,7 +14758,7 @@ static int bgp_clear_damp_route(struct vty *vty, const char *view_name,
 					if (pi->extra && pi->extra->damp_info) {
 						pi_temp = pi->next;
 						bgp_damp_info_free(
-							&pi->extra->damp_info,
+							pi->extra->damp_info,
```

Comment:
Do we need to set pi->extra->damp_info = NULL here now?

Reply:
You mean inside `bgp_damp_info_free()`?